cli/azd/pkg/project/framework_service_npm.go

```diff
@@ -130,11 +130,7 @@ func (np *npmProject) Package(
 	if err := buildForZip(
 		packageSource,
 		packageDest,
-		buildForZipOptions{
-			excludeConditions: []excludeDirEntryCondition{
-				excludeNodeModules,
-			},
-		}); err != nil {
+		buildForZipOptions{}); err != nil {
```

Comment:
These exclusions where moved to project_utils, because they are now dependent on the existance of .zipignore